node/src/main.rs

```diff
@@ -43,6 +43,9 @@ use graph_server_json_rpc::JsonRpcServer;
 use graph_server_websocket::SubscriptionServer as GraphQLSubscriptionServer;
 use graph_store_postgres::{Store as DieselStore, StoreConfig};
 
+const REORG_THRESHOLD: u64 = 50;
+const ANCESTOR_COUNT: u64 = 50;
+
```

Comment:
We could make these environmental variables... but I'll leave that to someone else ;)